### DISCUSSION TITLE
What does "Select Ticket Layout" do?

### DISCUSSION THREAD

**Discussion by archlinux-fan** [Leantime/leantime#550] [Q&A]
I have changed "Select Ticket Layout" in Project Settings from "Classic Tabs View" to "Modern One View".

I see no changes in To-Dos or tasks. 

Am I looking at the wrong place? Is it working yet?

**Comment by marcelfolaron** [comment]
It is a setting we added to allow for some design changes. The actual design change is not implemented right now. I'll take the AI to hide the setting for the time being

**Reply by archlinux-fan** [reply]
OK. Thank you for explanation.
